### PR TITLE
3.11 lightbox

### PR DESCRIPTION
**Pull request by yacon**

### CODE REVIEW

File: Kwf_js/EyeCandy/Lightbox/Lightbox.js [hunk 1]
```diff
@@ -358,39 +496,58 @@ Kwf.EyeCandy.Lightbox.Styles.Abstract.prototype = {
...
-                if ($(document.body).find('.lightboxMask').is(ev.target)) {
-                    if (Kwf.EyeCandy.Lightbox.currentOpen) {
-                        Kwf.EyeCandy.Lightbox.currentOpen.style.onMaskClick();
-                    }
-                }
-            });
+            var maskEl = $('<div class="kwfLightboxMask"></div>');
+            this.lightbox.lightboxEl.find('.kwfLightboxScrollOuter').append(maskEl);
+            setTimeout(function(){
+                $('.kwfLightbox').scrollTop(50000);
```
Comment (nsams) [inline Kwf_js/EyeCandy/Lightbox/Lightbox.js]: there can be multiple, use this.lightbox.lightboxEl

File: Kwf_js/EyeCandy/Lightbox/Lightbox.js [hunk 2]
```diff
@@ -14,7 +16,7 @@ $(document).on('click', 'a[data-kwc-lightbox]', function(event) {
         l = new Kwf.EyeCandy.Lightbox.Lightbox(href, options);
     }
     el.kwfLightbox = l;
-
+    this.kwfLightbox.style.fixContent();
```
Comment (nsams) [inline Kwf_js/EyeCandy/Lightbox/Lightbox.js]: unused, remove

File: tests/Kwc/Lightbox/LargeContent/Root.php [hunk 3]
```diff
@@ -11,6 +11,7 @@ public static function getSettings()
             'class' => 'Kwf_Component_Generator_Page_Static',
             'component' => 'Kwc_Lightbox_LargeContent_Test2_Component'
         );
+        $ret['assets']['dep'][] = 'ModernizrTouchevents';
```
Comment (nsams) [inline tests/Kwc/Lightbox/LargeContent/Root.php]: why touchevents?

File: tests/Kwc/Lightbox/Content/Root.php [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    {
+        $ret = parent::getSettings();
+        unset($ret['generators']['page']);
+        unset($ret['generators']['title']);
+        unset($ret['generators']['box']);
+        $ret['generators']['test'] = array(
+            'class' => 'Kwf_Component_Generator_Page_Static',
+            'component' => 'Kwc_Lightbox_Content_Test2_Component'
+        );
+        $ret['assets']['dep'][] = 'ModernizrTouchevents';
```
Comment (nsams) [inline tests/Kwc/Lightbox/Content/Root.php]: remove unused dependency

File: Kwf_js/EyeCandy/Lightbox/Lightbox.scss [hunk 5]
```diff
@@ -1,5 +1,208 @@
 @import "config/default-loading-animation-big";
+@import "config/lightbox-animation";
 @import "kwf/background-image-dpr2";
+@import "compass/css3/box-sizing";
+@import "compass/css3/transition";
+@import "compass/css3/transform";
+
+html.kwfup-lightboxActive, html.kwfup-lightboxActive body {
```
Comment (nsams) [inline Kwf_js/EyeCandy/Lightbox/Lightbox.scss]: kwfup

File: Kwf_js/EyeCandy/Lightbox/Lightbox.js [hunk 6]
```diff
@@ -253,26 +339,48 @@ Kwf.EyeCandy.Lightbox.Lightbox.prototype = {
...
+            }
         }
+        this.lightboxEl.addClass('kwfLightboxOpen');
         this.style.afterShow(options);
 
         Kwf.Statistics.count(this.href);
     },
     close: function(options) {
+        $('html').removeClass('kwfup-lightboxActive');
+        $('.kwfup-fixedContent').each(function(key, el){
```
Comment (nsams) [inline Kwf_js/EyeCandy/Lightbox/Lightbox.js]: what is fixedContent?

File: Kwf_js/EyeCandy/Lightbox/Lightbox.js (hunk 2 above)
Comment (yacon) [inline Kwf_js/EyeCandy/Lightbox/Lightbox.js]: What if a web has overwritten this method?

File: tests/Kwc/Lightbox/LargeContent/Root.php (hunk 3 above)
Comment (yacon) [inline tests/Kwc/Lightbox/LargeContent/Root.php]: done

File: tests/Kwc/Lightbox/Content/Root.php (hunk 4 above)
Comment (yacon) [inline tests/Kwc/Lightbox/Content/Root.php]: done

**Pull request merged**